demos/gopaste/snapcraft.yaml

```diff
@@ -18,13 +18,18 @@ parts:
     source: https://github.com/wisnij/gopaste.git
     source-type: git
   assets:
-    plugin: copy
+    plugin: dump
     source: https://github.com/wisnij/gopaste.git
     source-type: git
-    files:
-      static: .
-      web.template: .
+    snap:
+      - static
+      - web.template
   glue:
-    plugin: copy
-    files:
-      gopaste: bin/
+    plugin: dump
+    source: .
+    organize:
+        gopaste: bin/
+    filesets:
+        bin: [bin/gopaste]
```

Comment:
The spacing throughout seems to be two spaces, except for the organizes here, which look like four.